triton/apps/speech2text.rst

```diff
@@ -1,18 +1,252 @@
-speech2text: easy speech transcription
 ======================================
+Speech2Text: Easy Speech Transcription
+======================================
+
+Aalto speech2text app is available to researchers of Aalto University. 
+
+The app is run on the `Aalto Triton cluster <https://scicomp.aalto.fi/triton/>`__ and its usage is free for all Aalto researchers (master's thesis workers, doctoral students, post docs, visiting researchers). 
```

Comment:
Ah, I forgot that we just migrated to the same docs.

Although it seems to be :doc:`<../index>` instead of :doc:`<triton/index>` which makes it a bit less readable.